CreateInitialTables.sql

```diff
@@ -0,0 +1,244 @@
+
+INSERT INTO `images` (`image_id`, `route_id`, `url`, `alt_text`)
+VALUES (1,
+        1,
+        'https://images.unsplash.com/photo-1662363332703-2c5c4baa2b8d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80',
+        'Passo Dello Stelvio'
+        );
+INSERT INTO `images` (`image_id`, `route_id`, `url`, `alt_text`)
+VALUES (2,
+        2,
+        'https://cyclist.b-cdn.net/sites/cyclist/files/styles/gallery_adv/public/2018/07/hc_climb_alpe_dhuez_01.jpg?itok=SKrhfgYl',
+        "Alpe d''Huez"
+       );
+INSERT INTO `images` (`image_id`, `route_id`, `url`, `alt_text`)
+VALUES (3,
+        3,
+        'https://cyclist.b-cdn.net/sites/cyclist/files/styles/insert_main_wide_image/public/cheddar_gorge_035.jpg?itok=lQSbOnAY',
+        'Cheddar Gorge');
+INSERT INTO `images` (`image_id`, `route_id`, `url`, `alt_text`)
+VALUES (4,
+        4,
+        'https://alpsinsight.com/wp-content/uploads/2019/04/Cycling_Strade_Bianche_-1.jpg',
+        'Strade Bianchi'
+        );
+INSERT INTO `images` (`image_id`, `route_id`, `url`, `alt_text`)
+VALUES (5,
+        5,
+        'https://images.squarespace-cdn.com/content/v1/5d4428f4757d9c0001ab456c/99823340-88ef-4cdf-a0df-98da88cd4f0b/Bike+Hire+Loch+Lomond',
+        'Trossachs National Park'
+        );
+INSERT INTO `images` (`image_id`, `route_id`, `url`, `alt_text`)
+VALUES (6,
+        6,
+        'https://racing.trekbikes.com/wp-content/uploads/2021/05/image1-1.jpeg',
+        'Belgian Waffle Ride'
+        );
+INSERT INTO `images` (`image_id`, `route_id`, `url`, `alt_text`)
+VALUES (7,
+        7,
+        'https://h5d9a9f8.rocketcdn.me/wp-content/uploads/2018/02/Mountain-Bike-Tour-New-Zealand-HI-Adventures-Web-116.jpg',
+        'The Old Ghost Road'
+        );
+INSERT INTO `images` (`image_id`, `route_id`, `url`, `alt_text`)
+VALUES (8,
+        8,
+        'https://h5d9a9f8.rocketcdn.me/wp-content/uploads/2018/07/HI-Adventures-Mountain-Bike-Tour-Yukon-Low-54.jpg',
+        'Mountain Hero Mine Trail'
+        );
+INSERT INTO `images` (`image_id`, `route_id`, `url`, `alt_text`)
+VALUES (9,
+        9,
+        'https://h5d9a9f8.rocketcdn.me/wp-content/uploads/2018/08/HI-Adventures-Mountain-Bike-Tour-Morocco-Web-187.jpg',
+        'Tizi Mezzik Pass'
+        );
+INSERT INTO `images` (`image_id`, `route_id`, `url`, `alt_text`)
+VALUES (10,
+        10,
+        'https://keyassets.timeincuk.net/inspirewp/live/wp-content/uploads/sites/11/2013/06/rhyd-ddu-630x369.jpg',
+        'Rhyd Ddu'
+        );
+INSERT INTO `images` (`image_id`, `route_id`, `url`, `alt_text`)
+VALUES (11,
+        11,
+        'https://h5d9a9f8.rocketcdn.me/wp-content/uploads/2017/10/Mountainbike-Tour-Norway-Photo-Blog-27.jpg',
+        'Mefjellet'
+        );
+INSERT INTO `images` (`image_id`, `route_id`, `url`, `alt_text`)
+VALUES (12,
+        12,
+        'https://www.moredirt.com/trails/featured/bike-republic-solden.jpg',
+        'Galitzenklamm Gorge'
+        );
+
```

Comment:
Just one blank line at the end of a file is nice.

Reply:
Done